docs/index.html

```diff
@@ -205,7 +205,7 @@ <h2 class="text-green-500 text-lg title-font font-bold mb-2">
       </section>
     </div>
     <section class="text-gray-400 bg-gradient-to-br from-gray-900 to-gray-800 body-font">
-      <div class="container px-5 py-24 mx-auto">
+      <div class =  "px-5 py-24 mx-auto">
         <div class="flex flex-col text-center w-full mb-20">
           <h1 class="sm:text-3xl text-2xl font-bold title-font mb-4  text-green-500">Contributors</h1>
```

Comment:
This heading is smaller than the other headings. Make it a bit bigger to match the others.